### DISCUSSION TITLE
What does @tailwind components include by default?

### DISCUSSION THREAD

**Discussion by plmrry** [tailwindlabs/tailwindcss#9447] [Help] [answered]
By default, what is actually included by `@tailwind components`? It's hard to tell from the docs.

**Comment by wongjn** [comment, marked as answer]
Looking at the [source code](https://github.com/tailwindlabs/tailwindcss/blob/94d6e7299a7a4b9f94eb1698d7b1346c1572e797/src/corePlugins.js), only [Container](https://tailwindcss.com/docs/container).
